### ISSUE TITLE
Getting an error with MakeFile

### ISSUE THREAD

**Issue by UncleReaton** [bacardi55/gtl#2]
When I try to `make dependencies` I get this error:
```
CGO_ENABLED=0 go1.16.4 get "git.sr.ht/~adnano/go-gemini"
/bin/sh: line 1: go1.16.4: command not found
make: *** [Makefile:32: dependencies] Error 127
```
I know nothing about go but I fixed it by replacing on line 1 `go1.16.4` by `go`.

**Comment by bacardi55** [comment]
Erf, sorry my bad, I made it to specific for my system…
Could you simply edit the Makefile and replace
`CGO_ENABLED=0 go1.16.4`
with
`CGO_ENABLED=0 go`

edit: sorry, didn't read correctly your comment. Yes sorry, I use a specific go binary name and forgot to make it more standard for the makefile, I'll fix this, thanks!

**Comment by bacardi55** [comment]
Fixed in 9191e423d6229fadcaaad00843a2e79b2585cb3c
